"http://unblockproject.pw/","200","192.168.43.51","['54.253.102.29'; '3.104.250.127'; '3.106.21.213'; '3.24.209.200'; '52.63.243.35'; '13.211.116.246'; '3.24.247.184'; '52.63.187.77'; '54.206.145.91'; '3.104.147.29'; '3.105.225.164'; '13.210.128.241'; '3.106.169.116'; '3.106.67.141'; '3.106.37.199'; '13.55.191.56'; '52.64.155.227'; '3.24.37.110'; '54.253.238.204'; '54.153.204.20'; '54.79.190.247'; '54.206.220.49'; '3.105.240.211'; '52.64.227.94'; '52.63.120.34'; '3.24.151.179'; '54.253.148.58'; '54.79.252.3'; '13.55.216.130']","[403, ConnectionError(MaxRetryError(""HTTPConnectionPool(host='3.104.250.127', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD8290C8>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='3.106.21.213', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD844A08>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='3.24.209.200', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD84E288>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='52.63.243.35', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD851E48>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='13.211.116.246', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD85AA88>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='3.24.247.184', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD863688>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='52.63.187.77', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD86D288>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='54.206.145.91', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD872E48>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='3.104.147.29', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD87AA08>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), SSLError(MaxRetryError('HTTPSConnectionPool(host=\'3.105.225.164\', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname \'3.105.225.164\' doesn\'t match \'*.test.financefeast.io\'"")))')), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='13.210.128.241', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD896348>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='3.106.169.116', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD89E188>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='3.106.67.141', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD8A5188>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), 403, ConnectionError(MaxRetryError(""HTTPConnectionPool(host='13.55.191.56', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD8ACA48>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), 404, ConnectionError(MaxRetryError(""HTTPConnectionPool(host='3.24.37.110', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD8B2948>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), 451, 451, ConnectionError(MaxRetryError(""HTTPConnectionPool(host='54.79.190.247', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD8BBA08>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='54.206.220.49', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD8C5388>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='3.105.240.211', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD8CAF48>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='52.64.227.94', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD8D4B48>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='52.63.120.34', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD8DD748>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), ConnectionError(MaxRetryError(""HTTPConnectionPool(host='3.24.151.179', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001BBCD8E7348>: Failed to establish a new connection: [WinError 10060] A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond'))"")), 200, SSLError(MaxRetryError('HTTPSConnectionPool(host=\'54.79.252.3\', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(""hostname \'54.79.252.3\' doesn\'t match \'*.detectedx.com\'"")))')), 200]","['192.168.43.51', '10.194.29.145', '10.194.122.209', '172.24.126.1', '124.19.61.70', '52.95.217.190', '52.95.37.34', '150.222.112.243', '150.222.112.246', '52.95.38.222', '54.253.102.29', '52.95.37.73', '10.88.0.254', '54.253.102.29']","14","[]","[]","['162.210.196.166']","['192.157.56.140']","[]","[]","[200]","[200]"
